src/Vlingo.Wire.Tests/Fdx/Inbound/MockInboundStreamInterest.cs

```diff
@@ -30,15 +30,14 @@ public void HandleInboundStreamMessage(AddressType addressType, RawMessage messa
             var textMessage = message.AsTextMessage();
             TestResult.Messages.Add(textMessage);
             TestResult.MessageCount.IncrementAndGet();
-            Debug.WriteLine($"INTEREST: {textMessage} list-size: {TestResult.Messages.Count} count: {TestResult.MessageCount.Get()} count-down: {TestResult.UntilStops.Remaining}");
-            TestResult.UntilStops.Happened();
```

Comment:
You cannot just do that. You have to signal it has happened with something like `TestResult.UntilStops.WriteUsing(<name>, 1);` Look on how this is implemented in `MockChannelReaderConsumer`